### ISSUE TITLE
Please add these indices to Getbhavcopy

### ISSUE THREAD

**Issue by niteshmurti** [hemenkapadia/getbhavcopy#222]
Dear Hemen ji,

As you may be aware, NSE has permanently shut down their old website. As a result, Getbhavcopy is no longer working. Even after I went into Getbhavcopy Options and switched to the new website.

If you're working on a fix by creating a new version of Getbhavcopy, please also add these indices into the new version, as the current version of Getbhavcopy doesn't fetch the data for the following indices.

NIFTY Alpha Low-Volatility 30
NIFTY100 Quality 30
Nifty Smallcap 250
Nifty Smallcap 50
Nifty MidSmallcap 400
Nifty Midcap Select
NIFTY LargeMidcap 250
Nifty Oil & Gas
Nifty Financial Services 25/50
Nifty Healthcare Index
Nifty500 Multicap 50:25:25
Nifty Microcap 250
Nifty Total Market
Nifty India Digital
Nifty Mobility
Nifty India Defence
Nifty Financial Services Ex-Bank
Nifty Housing
Nifty Transportation & Logistics
Nifty MidSmall Financial Services
Nifty MidSmall Healthcare
Nifty MidSmall IT & Telecom
Nifty MidSmall India Consumption
Nifty Consumer Durables
Nifty Non-Cyclical Consumer
Nifty India Manufacturing
NIFTY100 Alpha 30
NIFTY200 Quality 30
NIFTY Midcap150 Quality 50
Nifty200 Momentum 30
Nifty100 ESG Sector Leaders
Nifty200 Alpha 30
Nifty Midcap150 Momentum 50
Nifty Smallcap250 Quality 50

I'm looking forward for a fix to this issue. Your Getbhavcopy software is very useful to all of us here... And since 31st March we've been manually updating the data on MetaStock/AmiBroker.

**Comment by touchofevil-dev** [comment]
I don't think this project is active development any longer. Planning to create a new version of the utility from scratch, and keep the source code available so it can have multiple hands contributing to it.
Repo - https://github.com/touchofevil-dev/BhavcopyEasy/

If anyone would like to help out with requests, please add it to the repo, and I will get this coded.
Please note, I may have limited time on my hand, so we may have to prioritize.

**Comment by pparesh25** [comment]
This Python program is a perfect alternative to the Getbhavcopy software. It generates the same text file as the one obtained by Getbhavcopy software. The generated file includes symbol, date, open, high, low, close and volume data arranged in columns and date format matches that of Getbhavcopy software. Additionally, all NSE index names have been changed to match the names used in Getbhavcopy's old files. This ensures compatibility with the old database of Amibroker or data import format without requiring any modifications.

https://github.com/pparesh25/Getbhavcopy-alternative
